submissions/madmaxWMFU/js-movie-booking/app.js

```diff
@@ -0,0 +1,156 @@
+const hall = document.querySelector('.hall');
+const cart = document.querySelector('.form__cart');
+const filmPoster = document.querySelector('.cinema-info__image');
+const filmTitle = document.querySelector('.description__title');
+const filmHall = document.querySelector('.description__hall');
+const filmDate = document.querySelector('.description__date');
+const filmTime = document.querySelector('.description__time');
+const emptyCart = document.querySelector('.tikets__wrap');
+const fullCart = document.querySelector('.tikets__form');
+const priceCart = document.querySelector('.form__total');
+
+const ticketPrice = {
+    "ordinary": 60,
+    "special": 30,
+    "comfort": 120
+};
+
+const getData = () => {
+    return fetch('./data/data.json')
+     .then(res => res.json())
+     .then((data) => data);
+}   
+
+const reverseDate = (date) => date.split("-").reverse().join("-");
+
+const renderData = (data) => {
+    filmPoster.src = data.poster;
+    filmTitle.innerText = `"${data.film}"`;
+    filmHall.innerText = `Зал №${data.hall}`;
+
+    const [date, time] = data.datetime.split(' ');
+    filmDate.innerText = reverseDate(date);
+    filmDate.setAttribute('datetime', date);
+
+    filmTime.innerText = time;
+    filmTime.setAttribute('datetime', time);
+
+    data.selected.forEach((item) => {
+        const seat = document.querySelector(`#seat${item.row}-${item.seat}`);
+        seat.closest('.hall__seat').classList.add('no-hover');
+        seat.setAttribute("disabled", "");
+        seat.nextElementSibling.classList.add(`${item.type}--active`);  
+    });
+}
+
+const renderCart = (ticket) => {
+    const [row, seat, price] = ticket.split('-');
+    const template = `
+        <div class="form__ticket" id="${ticket}">
+            <span class="form__ticket-row">Ряд: ${row}</span>    
+            <span class="form__ticket-seat">Місце: ${seat}</span>
+            <span class="form__ticket-price">Ціна: ${ticketPrice[price]}</span>
+        </div>`;
+    var ticketWrap = document.createElement("template");
+    ticketWrap.innerHTML = template;
+    cart.appendChild(ticketWrap.content);
+}
+
+const deleteFromCart = (id) => {
+    Array.from(cart.children).forEach((item) => {
+        if(item.getAttribute('id') === id) {
+           item.remove(); 
+        }
+    })
```

Comment:
```suggestion
    cart.querySelectorAll('.form__ticket').forEach((item) => {
        if(item.getAttribute('id') === id) {
           item.remove(); 
        }
    })
```

Use `querySelectorAll` instead `Array.form` he has included forEach method.

Reply:
Not fixed yet